public/app/features/admin/UpgradePage.tsx

```diff
@@ -217,7 +217,7 @@ const Item: React.FC<ItemProps> = ({ children, title, image }) => {
 
   return (
     <div className={itemStyle}>
-      <img src={imageUrl} />
+      <img src={imageUrl} alt="" />
```

Comment:
🤔 maybe the `alt` text can be the `${title} image`?

Reply:
I'm worried that might just be annoying for users who rely on screen readers, as it's basically just the same information twice (alt text, then title). I've just been thinking of the images here as list item decorations, but if anyone has an argument that they provide additional information or context, I'd be happy to change this.

Reply:
That's a very good point @kaydelaney, tbh I didn't know this was the way to implement [decorative images](https://www.w3.org/WAI/tutorials/images/decorative/) :)